src/containers/StudentDetailContainer/StudentDetailContainer.js

```diff
@@ -21,7 +21,8 @@ const StudentDetailContainer = ({ id }) => {
       <StudentDetailItem label='Язык:' info={studentById.language} />
       <StudentDetailItem label='Школа:' info={studentById.school} />
       <StudentDetailItem label='Контакты родителя(ей):' info={studentById.parentsContacts} />
-      <StudentDetailItem label='Направление:' info={studentById.streamId} />
+      <StudentDetailItem label='Направление:' info={studentById.Stream.name} />
```

Comment:
Программа упала при открытии подробной страницы студента

```suggestion
      <StudentDetailItem label='Направление:' info={studentById.Stream && studentById.Stream.name} />
```

Reply:
Исправлено